src/science/canopy/cable_canopy.F90

```diff
@@ -688,16 +689,34 @@ SUBROUTINE define_canopy(bal,rad,rough,air,met,dels,ssnow,soil,veg, canopy,clima
 
     canopy%cduv = canopy%us * canopy%us / (MAX(met%ua,CUMIN))**2
 
-    !---diagnostic purposes
-    canopy%gswx_T = rad%fvlai(:,1)/MAX( CLAI_THRESH, canopy%vlaiw(:) )         &
-         * canopy%gswx(:,1) + rad%fvlai(:,2) / MAX(CLAI_THRESH,     &
-         canopy%vlaiw(:))*canopy%gswx(:,2)
+! Evaluate Total Surface conductance !---diagnostic purposes
+
+!vlaiw is already limted?
+LAI_min(:)            = MAX( CLAI_THRESH, canopy%vlaiw(:) )
+Rel_bigLeafLAI_sun(:) = rad%fvlai(:,1) / LAI_min(:)
+Rel_bigLeafLAI_shd(:) = rad%fvlai(:,2) / LAI_min(:)
+
+!total stomatal conductance
+canopy_conductance(:) = Rel_bigLeafLAI_sun(:)  * canopy%gswx(:,1)              & 
+                      + Rel_bigLeafLAI_shd(:)  * canopy%gswx(:,2) 
 
-    ! The surface conductance below is required by dust scheme; it is composed from canopy and soil conductances
-    canopy%gswx_T = (1.-rad%transd)*MAX(1.e-06,canopy%gswx_T ) +  &   !contribution from  canopy conductance
-         rad%transd*(.01*ssnow%wb(:,1)/soil%sfc)**2 ! + soil conductance; this part is done as in Moses
-    WHERE ( soil%isoilm == 9 ) canopy%gswx_T = 1.e6   ! this is a value taken from Moses for ice points
+! The surface conductance below is required by dust scheme; it is composed from canopy and soil 
 
+! Canopy conductance
+minCanopyCond(:)      = MAX( 1.e-06, canopy_conductance(:) )
+canopy_conductance(:) = (1.-rad%transd(:))* minCanopyCond(:)
+
+! Soil conductance
+Rel_moisture(:) = ssnow%wb(:,1) / soil%sfc(:)
+soil_conductance(:) = rad%transd(:) * ( 0.01*Rel_moisture )**2 ! + soil conductance; this part is done as in Moses
+
+! Combined Surface conductance
+Surf_conductance(:) = canopy_conductance(:) + soil_conductance(:) 
+
+WHERE ( soil%isoilm == 9 ) Surf_conductance = 1.e6   ! this is a value taken from Moses for ice points
```

Comment:
Use ICE index here instead of hard-coded 9.
And can the proper indentation be added please. I think some lines are going beyond colum 80 as well

Reply:
fixed index and !comment over 80